System/Random.hs

```diff
@@ -68,57 +66,34 @@
 -- >>> fst $ stepGen $ snd $ stepGen (PCGen 17 29)
 -- 3288430965
 --
+-- Once implemented an instance of `RandomGen` can be created:
+--
 -- >>> :{
 -- instance RandomGen PCGen where
---   next g = (fromIntegral y, h)
---     where
---       (y, h) = stepGen g
+--   genWord32 = stepGen
```

Comment:
@idontgetoutmuch We don't want to promote implementation of deprecated methods in documentation. Documentation should always reflect how it should be done, now how it could be done. Therefore I removed instructions on how to create an instance for `RandomGen` with `next` and `genRange`.